.github/workflows/deploy-release.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         fileName: 'dhis_keystore.jks'
         encodedString: ${{ secrets.KEYSTORE }}
     - name: Build release apk
-      run: ./gradlew app:assembleRelease app:assembleDebug app:assemblePlayServices
+      run: ./gradlew app:assembleRelease app:assembleDhisDebug
```

Comment:
shouldn't be `assembleDhisRelease`?

Reply:
Nope. This generates all release apks (github and play store)